db/dcrpg/pgblockchain.go

```diff
@@ -711,23 +720,23 @@ func (pgb *ChainDB) HeightHashDB() (uint64, string, error) {
 }
 
 // Height uses the last stored height.
-func (pgb *ChainDB) Height() uint64 {
```

Comment:
I'm surprised losing (*ChainDB).Height/Hash didn't affect more code, namely callers from other packages.

To stabilize the `ChainDB` API a bit, we could add some wrappers for the removed functions, but this losing Height and Hash might be a good thing since it wasn't completely clear when to use Height vs. HeightDB, and Hash vs. HashDB.  So anyway, this looks good.